clients/admin-ui/src/features/datastore-connections/add-connection/constants.ts

```diff
@@ -38,6 +39,10 @@ export const CONNECTOR_PARAMETERS_OPTIONS: ConnectorParameterOption[] = [
       ConfigurationSettings.DSR_CUSTOMIZATION,
     ],
   },
+  {
+    type: SystemType.EMAIL,
+    options: [ConfigurationSettings.CONNECTOR_PARAMETERS],
+  },
```

Comment:
@allisonking I accidentally commited some front end pieces I was experimenting with here as I was trying to understand what was needed. I can remove if you'd like or let you adjust further as needed.

Reply:
no worries, go ahead and leave it and I'll clean it up!

Reply:
awesome thank you